### ISSUE TITLE
nm-subscriber: syntax.c:1133: syntax_skip_prefix: Assertion `lcp <= a->length' failed.

### ISSUE THREAD

**Issue by arr2036** [alandekok/recli#22]
```text
nm-subscriber> show client id
nm-subscriber: syntax.c:1133: syntax_skip_prefix: Assertion `lcp <= a->length' failed.
==2911== 
==2911== HEAP SUMMARY:
==2911==     in use at exit: 16,815,907 bytes in 620 blocks
==2911==   total heap usage: 4,204 allocs, 3,584 frees, 33,714,471 bytes allocated
==2911== 
==2911== LEAK SUMMARY:
==2911==    definitely lost: 0 bytes in 0 blocks
==2911==    indirectly lost: 0 bytes in 0 blocks
==2911==      possibly lost: 0 bytes in 0 blocks
==2911==    still reachable: 16,815,907 bytes in 620 blocks
==2911==         suppressed: 0 bytes in 0 blocks
==2911== Reachable blocks (those to which a pointer was found) are not shown.
==2911== To see them, rerun with: --leak-check=full --show-leak-kinds=all
==2911== 
==2911== For counts of detected and suppressed errors, rerun with: -v
==2911== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
Aborted (core dumped)
```

**Comment by arr2036** [comment]
It's just ``show client id`` entering the context of ``show client`` is fine.

**Comment by alandekok** [comment]
I can reproduce this.

**Comment by alandekok** [comment]
Fixed by commit 9637019
